### PR TITLE
V0.5.0 new

### PR DESCRIPTION
**Pull request by marciofunes**
Hello, PR with some PO reviews and corrections :D

### CODE REVIEW

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 1]
```diff
@@ -37,416 +40,430 @@ msgstr "Requisitos de hardware"
...
+msgstr "Para a instalação com kubernetes por favor, verifique o pdf abaixo"
 
 #: ../../source/installation-guide.rst:210
 msgid ""
-"click here to access the dojot installation guide with kubernetes"
-msgstr ""
-"Clique aqui para acessar o guia de instalação da dojot com kubernetes"
+":download:`click here to access the dojot installation guide with kubernetes "
+"<pdf/Dojot-Installation-Guide.pdf>`"
+msgstr "Clique aqui para acessar o guia de instalação da dojot com kubernetes"
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:452]: Missing link in translation

File: source/locale/pt_BR/LC_MESSAGES/load-testing-dojot-platform.po [hunk 2]
```diff
@@ -72,698 +71,740 @@ msgid "Running a simple test"
 msgstr "Rodando um teste simples"
 
 #: ../../source/load-testing-dojot-platform.rst:38
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/load-testing-dojot-platform.po:74]: Resolve fuzzy

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 3]
```diff
@@ -112,58 +117,72 @@ msgstr "Existem 3 repositórios principais:"
...
-"`<https://github.com/dojot/docker-compose>`_: repository for Docker compose "
-"files and configurations. This is what we would recommend to use to start with."
+"`<https://github.com/dojot/docker-compose>`_: repository with the files "
+"and settings"
+msgstr ""
+"`<https://github.com/dojot/ansible-dojot>`_: repositório com os arquivos "
+"e configurações para executar a dojot em ambiente *Kubernetes*."
+
+#: ../../source/faq/faq.rst:50
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/faq.po:137]: Resolve fuzzy

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po [hunk 4]
```diff
@@ -7,46 +7,51 @@ msgid ""
...
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.8.0\n"
+"Generated-By: Babel 2.9.0\n"
 
 #: ../../source/using-web-interface.rst:2
 msgid "Using web interface"
 msgstr "Usando a interface WEB"
 
 #: ../../source/using-web-interface.rst:4
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:26]: Resolve fuzzy

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po [hunk 5]
```diff
@@ -126,19 +133,20 @@ msgstr ""
 "dispositivo baseado em ambos os modelos foi criado e um novo atributo "
 "também chamado ``level`` foi adicionado ao modelo ``ThermTemplate``."
 
-#: ../../source/using-web-interface.rst:74
+#: ../../source/using-web-interface.rst:75
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:137]: Resoslve fuzzy

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po [hunk 6]
```diff
@@ -217,24 +225,23 @@ msgid "Flow configuration"
 msgstr "Configuração de fluxo"
 
 #: ../../source/using-web-interface.rst:125
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:228]: Resolve fuzzy

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po [hunk 7]
```diff
@@ -520,53 +528,57 @@ msgid "Performing access to the Dashboard"
 msgstr "Realizando acesso ao Dashboard"
 
 #: ../../source/using-web-interface.rst:286
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:531]: Resolve fuzzy

**Review by mprevide — CHANGES_REQUESTED** [review]
I left some comments.
Please always check if there are fuzzies

**Comment by mprevide** [comment]
There is a conflict, check that

File: source/locale/pt_BR/LC_MESSAGES/load-testing-dojot-platform.po (hunk 2 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/load-testing-dojot-platform.po:74]: Done

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 3 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/faq.po:137]: Done

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po (hunk 4 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:26]: Done

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po (hunk 5 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:137]: Done

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po (hunk 6 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:228]: Done

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po (hunk 7 above)
Comment (marciofunes) [reply source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:531]: Done

**Comment by mprevide** [comment]
There is more conflits...

**Comment by mprevide** [comment]
I think You should create a branch based in release/v0.5.1 and add just the new changes

**Comment by mprevide** [comment]
Or you can create this pr to release/v0.5.2, but doesnt forget about using the right base.

**Comment by mprevide** [comment]
Please, refer a issue in this PR

**Pull request closed without merging**